test/canaries/deploy_canaries.yml

```diff
@@ -0,0 +1,39 @@
+---
+- hosts: all
+  become: true
+  roles:
+    - role: caos.ansible_roles.docker_install
+    - role: pip
+      vars:
+        pip_install_packages:
+          - name: docker
+            extra_args: --ignore-installed
+          - name: docker-compose
+
+- name: Run containers with the provided images
+  hosts: all
+  gather_facts: true
+  become: true
+  tasks:
+    - community.docker.docker_compose:
+        project_name: otel-collector_canaries
+        definition:
+          version: '3'
+          services:
+            previous_collector:
```

Comment:
what configuration will be applied? 🤔

Reply:
The default one: https://github.com/newrelic/opentelemetry-collector-releases/blob/main/distributions/nr-otel-collector/Dockerfile#L13

But we still don't have a configuration with NR env vars (on going https://github.com/newrelic/opentelemetry-collector-releases/pull/16)

Reply:
👍 I was thinking on having an external conf mounted in `/etc/nr-otel-collector/config.yaml` in docker-compose, but it makes sense, lets do it in the ongoing task

Reply:
Great! Let's evaluate it once we have the release and a default config. I think it would be great to use the default configuration, so we test the same installation environment.